ArchiSteamFarm/BotConfig.cs

```diff
@@ -101,6 +101,9 @@ internal sealed class BotConfig {
 		internal readonly ulong SteamMasterID = 0;
 
 		[JsonProperty]
+		internal readonly ulong SteamOperatorID = 0;
```

Comment:
We're missing default entry in `example.json`, as well as addition in `ConfigGenerator/BotConfig.cs`.